bus-mapping/src/evm/opcodes/mload.rs

```diff
@@ -67,6 +66,7 @@ mod mload_tests {
         evm::StackAddress,
         mock,
     };
+    use pretty_assertions::assert_eq;
```

Comment:
Move at the top of the file if possible for consistency.

Reply:
Hm, this is a dev dependency only used in the test.

Reply:
My bad... :disappointed: